Eastern-Research-Group/Trade-off_Tool_for_Sampling
# Feature/103 switch us structures

## Related Issues:
* [TOTS-103](https://ergcloud.atlassian.net/browse/TOTS-103)

## Main Changes:
* Switched to USA Structures layer (used to use NSI) to power the decon tool.

## Steps To Test:
1. Create a decon plan
2. Run the tool
3. Verify the buildings shown are polygons of the building footprint instead of icons
4. Verify the legend looks correct
